pkg/utils/dataset/volume/delete.go

```diff
@@ -17,19 +17,35 @@ func DeleteFusePersistentVolume(client client.Client,
 	runtime base.RuntimeInfoInterface,
 	log logr.Logger) (err error) {
 
-	found, err := kubeclient.IsPersistentVolumeExist(client, runtime.GetName(), common.ExpectedFluidAnnotations)
+	deprecated, err := HasDeprecatedPersistentVolumeName(client, runtime, log)
```

Comment:
Why do we need to check if pv name is deprecated when creating pv?

Reply:
What's the logic when it's found there is an existing pv during pv creation?

Reply:
The check is to avoid duplicate pv creation when there is an existing pv. 

When there is an existing pv, the PR leaves it there and do nothing until the dataset is destroyed

Reply:
I'm thinking if it's a bug if we do nothing on it.